var/spack/repos/builtin/packages/h5utils/package.py

```diff
@@ -14,6 +14,7 @@ class H5utils(AutotoolsPackage):
     url      = "http://ab-initio.mit.edu/h5utils/h5utils-1.12.1.tar.gz"
     list_url = "http://ab-initio.mit.edu/h5utils/old/"
 
+    version('1.13.1', sha256='c5a76f064d6daa3e65583dce2b61202510e67cf6590f076af9a8aa72511d7d65')
```

Comment:
I cannot confirm the sha256.  I'm getting "All fetchers failed".

Reply:
@tldahlgren Fixed "spack checksum": Added new commit: make 'spack checksum' work for current and future versions